README.md

```diff
@@ -89,6 +89,10 @@ Of course there are more [examples for you to peruse](examples/). You could also
 2. `script/release`
 3. Profit.
 
+### Development in docker
```

Comment:
I know basically nothing about docker. 😬  Would you be willing to be a collaborator on the repo, answer any docker questions and review any docker PRs that come up?